wire/message_test.go

```diff
@@ -34,6 +35,16 @@ func makeHeader(kaspaNet KaspaNet, command string,
 	return buf
 }
 
+func idFromBytes(idBytes []byte) *id.ID {
```

Comment:
Maybe move this method to `id` package?
Sounds like someone somewhere will need it.